src/backend/app_chooser.rs

```diff
@@ -143,15 +142,14 @@ impl AppChooserInterface {
 
     async fn update_choices(
         &self,
-        #[zbus(object_server)] server: &ObjectServer,
         handle: OwnedObjectPath,
         choices: Vec<AppID>,
     ) -> Result<(), PortalError> {
         #[cfg(feature = "tracing")]
         tracing::debug!("AppChooser::UpdateChoices");
 
-        let iface_ref = server.interface::<_, Request>(handle).await?;
```

Comment:
Something that might be worth adding, is two methods to HandleToken that gives you an InterfaceRef<Request> or InterfaceRef<Session>, taking care of the conversion back from a HandleToken to an ObjectPath, what do you think?